src/flyte/_internal/imagebuild/docker_builder.py

```diff
@@ -621,7 +593,8 @@ async def _ensure_buildx_builder():
         else:
             logger.info("Buildx builder already exists.")
 
-    def get_docker_ignore(self, image: Image) -> Optional[Path]:
+    @staticmethod
+    def get_docker_ignore(image: Image) -> Optional[Path]:
```

Comment:
Could we move this function to `imagebuild/utils.py`, so the remote builder can use it in the future?